.travis.yml

```diff
@@ -0,0 +1,27 @@
+language: generic
+os: linux
+dist: bionic
+services: docker
+
+script:
+- sudo apt-get install texlive-latex-extra texlive-lang-polish texlive-generic-recommended texlive-fonts-recommended texlive-fonts-extra pandoc make
```

Comment:
Maybe move this to `install` instead of `script`? Purely cosmetic change. See: https://docs.travis-ci.com/user/job-lifecycle/